src/GTP.cpp

```diff
@@ -875,10 +899,158 @@ bool GTP::execute(GameState & game, std::string xinput) {
         } else {
             gtp_fail_printf(id, "syntax not understood");
         }
+        return true;
+    } else if (command.find("lz-memory_report") == 0) {
+        auto base_memory = (size_t)(s_network->get_estimated_size() *
+            Network::CONSERVATIVE_MEMORY_OVERHEAD);
+        auto tree_size = UCTNodePointer::get_tree_size();
+        auto cache_size = s_network->get_estimated_cache_size();
+
+        auto total = base_memory + tree_size + cache_size;
+        printf("Estimated total memory consumption: %d MiB.\n", total / MiB);
```

Comment:
This probably needs gtp_printf, make sure the output is legal according to the GTP spec?

Reply:
Converted to gtp_printf.
I think I now follow the GTP spec regarding multiline responses.